R/util.R

```diff
@@ -1429,3 +1429,24 @@ excel_numeric_to_date <- function(date_num, date_system = "modern",
   janitor::excel_numeric_to_date(as.numeric(date_num), date_system = date_system,
                                  include_time = include_time, round_seconds = round_seconds)
 }
+
+#' Workaround for toJSON output getting affected by SJIS damemoji characters
+#' when LC_CTYPE is set to CP932 (SJIS).
+#' Switch LC_CTYPE to CP1252 before calling jsonlite::toJSON, 
```

Comment:
What is CP1252? Why we need to switch to 1252? It would be great if you can describe briefly in the comment too.